mobile/src/modules/auth/screens/HomeScreen.tsx

```diff
@@ -104,9 +107,24 @@ class Home extends React.PureComponent<TProps, State> {
 
   render () {
     const accounts: Account[] = this.props.auth.accounts || [];
+    const totalBalance = accounts.map(({ balanceNQT }) => balanceNQT)
```

Comment:
Maybe don't use map and reduce as shorthand to our code, because it's root view and it should be fast? But it's 1-6 accounts in general, so i don't think it's a problem or so

but

`curr.balanceNQT` and we don't need another mapped array.